package.json

```diff
@@ -44,22 +45,22 @@
     "bridgeStandardToken": "ts-node scripts/deployStandard.ts"
   },
   "dependencies": {
-    "@arbitrum/nitro-contracts": "^1.0.0-beta.2",
-    "@arbitrum/sdk-classic": "npm:@arbitrum/sdk@1.1.4",
-    "@arbitrum/sdk-nitro": "npm:@arbitrum/sdk@3.0.0-beta.0",
     "@ethersproject/address": "^5.0.8",
     "@ethersproject/bignumber": "^5.1.1",
     "@ethersproject/bytes": "^5.0.8",
     "@typechain/ethers-v5": "9.0.0",
     "@types/prompts": "^2.0.14",
     "@types/yargs": "^17.0.9",
-    "arb-bridge-peripherals": "1.0.9",
     "dotenv": "^10.0.0",
     "ethers": "^5.1.0",
     "ts-node": "^10.2.1",
     "typechain": "7.0.0"
   },
   "devDependencies": {
+    "arb-bridge-peripherals": "1.0.9",
+    "@arbitrum/nitro-contracts": "^1.0.0-beta.2",
+    "@arbitrum/sdk-classic": "npm:@arbitrum/sdk@1.1.5",
```

Comment:
This and the sdk-nitro need to be in prod deps, since we call them directly in this package.